docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -19,6 +19,24 @@ Since v6 is a major release, it contains some changes that affect the public API
 These changes were done for consistency, improve stability and make room for new features.
 Below are described the steps you need to make to migrate from v5 to v6.
 
+### Drop `clock` in desktop mode
+
+We have dropped the clock picker behavior in desktop mode for `DateTimePicker` and `TimePicker` components.
```

Comment:
@joserodolfofreitas or any1 else—feel free to suggest improvements on this or the below sections. 🙏

Reply:
In desktop mode, the `DateTimePicker` and `TimePicker` components will not display the clock.

Reply:
Updated 😉